packages/@adobe/react-spectrum/test/ActionGroup.test.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {ActionGroup, Item} from '../actiongroup';
+import {Provider} from '@react-spectrum/provider';
+import React from 'react';
+import {render} from '@testing-library/react';
+import scaleMedium from '@adobe/spectrum-css-temp/vars/spectrum-medium-unique.css';
```

Comment:
should be able to import the default theme instead of creating it yourself
https://github.com/adobe-private/react-spectrum-v3/blob/master/packages/@react-spectrum/theme-default/src/index.ts
it's in this package https://github.com/adobe-private/react-spectrum-v3/pull/515/files#diff-5e42d4a66aee6edf6851d6697b1bfadcR41
so probably can just `'../theme-default'`